packages/console/src/command-history/CommandHistory.test.jsx

```diff
@@ -38,176 +46,192 @@ function mountItems(items) {
       commandHistoryStorage={{ addItem() {}, updateItem() {}, getTable() {} }}
     />
   );
-
-  wrapper.setState({
-    items,
-    itemCount: items.length,
-    offset: 0,
-  });
+  const items = makeItems(itemLength);
+  updateFunction({ items, offset: 0 });
 
   return wrapper;
 }
 
-function getItem(wrapper, itemIndex) {
-  return wrapper.find('div.command-history-item').at(itemIndex);
+function clickItem(itemIndex, mouseEventInit = {}) {
+  const item = screen.getByText(`Command ${itemIndex}`);
+  fireEvent.mouseDown(item, mouseEventInit);
+  fireEvent.mouseUp(item, mouseEventInit);
 }
 
-function clickItem(wrapper, itemIndex, mouseEventInit = {}) {
-  getItem(wrapper, itemIndex).simulate('mousedown', mouseEventInit);
-
-  getItem(wrapper, itemIndex).simulate('mouseup', mouseEventInit);
-}
-
-function dragRange(wrapper, start, end, mouseEventInit = {}) {
-  getItem(wrapper, start).simulate('mousedown', mouseEventInit);
+function dragRange(start, end) {
+  const startItem = screen.getByText(`Command ${start}`);
+  fireEvent.mouseDown(startItem);
   for (let i = start; i <= end; i += 1) {
-    getItem(wrapper, i).simulate('mousemove', mouseEventInit);
+    const item = screen.getByText(`Command ${i}`);
+    fireEvent.mouseMove(item);
   }
-  getItem(wrapper, end).simulate('mouseup', mouseEventInit);
+  const endItem = screen.getByText(`Command ${start}`);
+  fireEvent.mouseUp(endItem);
 }
 
 it('renders an empty list without crashing', () => {
-  mountItems([]);
+  mountItems(0);
 });
 
 it('renders a list with items without crashing', () => {
-  mountItems(defaultItems);
+  mountItems();
 });
 
-it('handles selecting an item on click', () => {
-  const wrapper = mountItems(defaultItems);
+function expectSelected(index) {
+  expect(screen.getByText(`Command ${index}`).parentNode.className).toContain(
+    'active'
+  );
```

Comment:
Use `.toHaveClass` instead (like in `ChartBuilder.test.jsx`:
```suggestion
  expect(screen.getByText(`Command ${index}`).parentNode).toHaveClass('active');
```